src/texstudio.cpp

```diff
@@ -110,7 +110,7 @@
  */
 
 
-
+QActionGroup *previewModeGroup = nullptr;
```

Comment:
there is no need to use a global variable here.
global variables are considered bad in cpp.